README.md

```diff
@@ -116,12 +119,18 @@ Let's say that you just blindly committed changes with `git commit -a` and you'r
 (master)$ git show
 ```
 
-or
+Or
 
 ```sh
 $ git log -n1 -p
 ```
 
+If you want to see a file at a specific commit, you can also do this:
+
+```sh
+$ git show commitid:filename
```

Comment:
I know we have a lot of ways of representing "a commit id" but personally I prefer something a little more distinguishable from an actual command (like `HASH1234` or `b7c028`) or explicit clarification in the description, i.e. `you can also do this (where <commitid> is the commit you're interested in):`

Reply:
This is part of a larger issue. We should add a section at the top with conventions for the document, perhaps? For instance, we could use a nonce SHA (`b7c028`) throughout the document whenever we mean a commit id.